browser/ui/BUILD.gn

```diff
@@ -1351,8 +1353,8 @@ source_set("ui") {
 source_set("unit_tests") {
   if (!is_android) {
     testonly = true
-    sources = []
-    deps = []
+    sources = [ "brave_scheme_utils_unittest.cc" ]
+    deps = [ "//testing/gtest" ]
```

Comment:
```suggestion
    deps = [
      "//chrome/browser/ui",
      "//testing/gtest",
     ]
```

Reply:
Sorry, I thought I have already pushed it.

Pushed now, PTAL.